mailpoet/lib/Config/MP2Migrator.php

```diff
@@ -134,7 +134,7 @@ private function tableExists($table) {
     global $wpdb;
 
     try {
-      $sql = "SHOW TABLES LIKE '{$table}'";
+      $sql = $wpdb->prepare("SHOW TABLES LIKE %s", esc_sql($table));
```

Comment:
I think `esc_sql()` is not necessary here.

Reply:
My problem is that using `prepare()` and `esc_sql()` in combination results in escaping the value twice with `mysqli_real_escape_string()`